web/ui/karma.conf.js

```diff
@@ -96,10 +95,10 @@ module.exports = function(config) {
     // Code coverage results below these thresholds will trigger a build failure
     // For more info, see https://www.npmjs.com/package/karma-threshold-reporter
     thresholdReporter: {
-      statements: 5,
-      branches: 0,
-      functions: 5,
-      lines: 5
+      statements: 8,
```

Comment:
Looks like these can be tightened even further to something like:
```
   statements: 17,
   branches: 10,
   functions: 14,
   lines: 17
``` 

In practice, we might want the thresholds to stay within -2 of the latest high-water mark rather than -1 like I have above.  On one hand, we should strive to increase coverage slowing over time, but minor additions shouldn't trigger failures.  In the other words, the values I suggested might be too tight, but there's certainly room to increase them over what's there now.